cabal-install/Distribution/Client/ProjectOrchestration.hs

```diff
@@ -483,6 +484,14 @@ printPlan verbosity
     showFlagValue (f, False)  = '-' : showFlagName f
     showFlagName (PD.FlagName f) = f
 
+    showProfiling elab =
+        (if elabProfLib elab
+            then " profile"
+            else "") ++
+        (if elabProfExe elab
+            then " exe-profile"
+            else "")
```

Comment:
It's horrible forever.

"profile" means that we will pass `--enable-lib-profiling` to configure, and "exe-profile" means we'll pass `--enable-exe-profiling` to configure. Whether or not these flags *do* anything is a separate matter. ;)